ontotext-yasgui-web-component/src/services/yasqe/yasqe-service.ts

```diff
@@ -45,4 +50,14 @@ export class YasqeService {
       () => this.eventService.emit(InternalCreateSavedQueryEvent.TYPE, new InternalCreateSavedQueryEvent()));
     return buttonElement;
   }
+
+  private buildShareQueryButton(): HTMLElement {
+    const buttonElement = document.createElement("button");
+    buttonElement.className = "yasqe_shareQueryButton custom-button icon-link";
+    buttonElement.title = this.translationService.translate('yasqe.actions.share_query.button.tooltip');
+    buttonElement.setAttribute("aria-label", this.translationService.translate('yasqe.actions.share_query.button.tooltip'));
```

Comment:
We have to think about re-translate labels when language is changed.

Reply:
Will think about it with the next actions